src/dashboard/install/README.md

```diff
@@ -499,6 +499,21 @@ These variables specify the behaviour of LDAP authentication. Only applicable if
     - **Type:** `string`
     - **Default:** ``
 
+- **`AUTH_LDAP_USER_ATTR_FIRST_NAME`**:
+    - **Description:** LDAP attribute for first name.
+    - **Type:** `string`
+    - **Default:** `givenName`
+
+- **`AUTH_LDAP_USER_ATTR_LAST_NAME`**:
+    - **Description:** LDAP attribute for last name.
+    - **Type:** `string`
+    - **Default:** `sn`
+
+- **`AUTH_LDAP_USER_ATTR_EMAIL`**:
```

Comment:
Since your new settings represent attributes, what do you think of renaming them to: `AUTH_LDAP_USER_FIRST_NAME_ATTR`, `AUTH_LDAP_USER_LAST_NAME_ATTR`, `AUTH_LDAP_USER_EMAIL_ATTR`?


Looking at the other settings in this section I notice they end in terms like `..._DN`, `..._FILTERSTR`, `..._TEMPLATE` which describe their meaning a bit better.

Reply:
Renamed as recommended.